chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/pendingListing/reviewer/edition2015/duplicate/TestData2015DuplicateReviewer.java

```diff
@@ -37,7 +38,7 @@ public void review(final PendingCertifiedProductDTO listing, final PendingCertif
         if (testDataDuplicateResults.duplicatesExist()) {
             listing.getWarningMessages().addAll(
                     getWarnings(testDataDuplicateResults.getDuplicateList(),
-                            certificationResult.getCriterion().getNumber()));
+                            Util.formatCriteriaNumber(certificationResult.getCriterion())));
```

Comment:
I know `Util.formatCriteriaNumber(...)` is all over the place and not necessarily part of this change, I do not like this method being in the `Util` class.  
http://ralin.io/blog/oop-anti-patterns-utility-or-helper-classes.html

Is there a way we can get this to be a part of the `CertificationResult`, `CertificationResultDTO` classes?  I see two different methods in `Util` that do the same thing to strings in different objects.

Reply:
I originally added it in CertificationCriterion domain object. Then Dan moved it into Util to have one method for each of those two types. I can refactor it back...